src/app/screens/BookDetail/components/Suggestions/layout.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import defaultImage from '../../../../../assets/ASSETS/default_book.svg';
+import './styles.css';
+
+function SuggestionsLayout({ books }) {
+  return (
+    <div className="suggestions-container">
+      <h1 className="suggestions-title">Sugerencias</h1>
+      <div className="suggestions-books">
+        {books.map(book => (
+          <div key={book.id}>
+            {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+            <Link to={`/books/${book.id}`}>
+              {book.image_url ? (
+                <img className="suggestions-image" src={book.image_url} alt={book.title} />
+              ) : (
+                <div className="suggestions-image">
+                  <img src={defaultImage} className="suggestions-image-default" alt={book.title} />
+                </div>
```

Comment:
this code is repeated in a lot of places. Think a way of removing duplicated code 😄